receiver/mongodbatlasreceiver/internal/metadata/generated_metrics.go

```diff
@@ -492,7 +492,9 @@ var Metrics = &metricStruct{
 			metric.SetName("mongodbatlas.process.db.document.rate")
 			metric.SetDescription("Document access rates")
 			metric.SetUnit("{documents}/s")
-			metric.SetDataType(pdata.MetricDataTypeGauge)
+			metric.SetDataType(pdata.MetricDataTypeSum)
+			metric.Sum().SetIsMonotonic(true)
+			metric.Sum().SetAggregationTemporality(pdata.MetricAggregationTemporalityCumulative)
```

Comment:
You're right - I was thinking I needed to use this to calculate the rate, but these metrics are already reporting the rate. I think leaving as gauge is fine. Thanks for the quick feedback 😄

Reply:
Id bugged if I have given an accurate advice here. There's a concept called "non-monotonic sum" which can be used with delta aggregation temporality. I did some more research and also [asked on Slack](https://cloud-native.slack.com/archives/C01NP3BV26R/p1645381172080209). As @Aneurysm9 noted, this allows to do the aggregation when a dimensions is removed. I looked at dimensions for this metrics (on top of resource dimensions there's `document_status` which might be `returned, inserted, updated, deleted`).

I am not fully sure but I _think_ that in such case using non-monotonic sum might actually work since document/s rates over various dimensions could be aggregated together, if needed?

Reply:
@pmm-sumo a "rate" is not a "Sum" in the first place. So I am confused about your confusion for this being "non-monotonic sum".

Reply:
I probably overthinked this and also missed providing more details. I was deliberating if current document/s reading could be replaced with delta (so here, change in number of documents/s, which can be easily calculated) non-monotonic sum could work and perhaps it would be a better fit

I get it that it's not, apologies for the confusion